README.md

```diff
@@ -19,6 +19,7 @@ The redis source has 2 modes
 
 ### MODE_PUBSUB
 
+haris: here it says "provided in key". what's this key? how is it different from mode?
```

Comment:
Redis is a key-value store, and we are listening for changes on Stream keys and Pub/Sub channels.
Mode is basically the type of data that key holds.
The Key is the data that we are actually subscribing to.